azerothcore/azerothcore-wotlk
# [Worldserver] Increased limit of realms

### More than 8 realms available
Thanks @Rochet2 for all the help
##### CHANGES PROPOSED:

-  Altered the DB and the core to allow more than 8 realms.
With these changes the core will support until 255 available realms.
Due to the client only reading a package with a size of 8 bits, the core can't allow more realms to be added since any higher values won't be updated in the database. Otherwise, the server could behave abnormally because we won't be storing the current realm onto which a player is logged on.

###### ISSUES ADDRESSED:


Closes #1403 


##### TESTS PERFORMED:


I tested this on windows and it works.


##### HOW TO TEST THE CHANGES:


1. Add a new realm on Auth Database -> Realmlist with ID 9 or higher (Also need diferent name)
2. Change the realm id on worldserver.conf to the new value that you added on the database
3. Launch worldserver and log onto the realm
4. Check worldserver for no query errors
5. On the auth database at accounts.online you will see the number of the realmID to which the player is connected to, in this case it should display 9.


##### Target branch(es):

Master
